shared/tasks/browser.mjs

```diff
@@ -32,17 +32,29 @@ export async function screenshots () {
   const browser = await launch()
   const componentNames = await getDirectories(join(paths.src, 'govuk/components'))
   const exampleNames = ['text-alignment', 'typography']
+  const rebrandExamples = [
+    {
+      component: 'header',
+      example: 'full-width-with-navigation'
```

Comment:
I think it'd be useful to also capture a screenshot for 'Header with service name and navigation'. Might be worth allowing for an array of `examples` here instead?

I also think it'd be useful to capture screenshots of the two additional examples _without_ the rebrand enabled, so we can make sure we don't accidentally change them.

Reply:
> I also think it'd be useful to capture screenshots of the two additional examples without the rebrand enabled, so we can make sure we don't accidentally change them.

I might be missing something, I don't think this has been done yet.

Reply:
It has now. I was just trying out a solution for the first bit of your comment on the PR itself, settle down 😂